.gitignore

```diff
@@ -3,13 +3,12 @@
 /local.properties
 /.idea/workspace.xml
 /.idea/libraries
-/.idea/gradle.xml
-/.idea/modules.xml
 settings.gradle
 /.idea/caches/**
+/demowebsocket/**
```

Comment:
Do we need this here in this PR?

Reply:
I need to include this to every PR until the WebSocket PR is merged, then I can remove it

Reply:
Fixed